schedule.yaml

```diff
@@ -25,11 +25,13 @@ tabs:
   - title: 'Tutorial: SnowEx Mission Overview'
     description: All about SnowEx!
     time: 11:15 - 12:00
+    tag: snowex
```

Comment:
could this be 'label' instead of tag and https://github.com/uwhackweek/pyhackweek?tab=readme-ov-file#hw_create_schedule could then just add copy them into schedule.yaml

Reply:
We can name that to our liking. the `hw_create_schedule` tool manually maps issue information and we have a lot of flexibility with that.